pubspec.yaml

```diff
@@ -39,3 +39,11 @@ dev_dependencies:
   mockito: ^4.0.0
   over_react_test: ^2.4.0
   test: ^1.0.0
+
+
+dependency_overrides:
+  react:
+    git:
+      url: https://github.com/cleandart/react-dart.git
+      ref: c2eec28f0cf4189e15a61a49eef8863b77111924 # from 5.0.0-wip
```

Comment:
#nit can we change this ref to just `5.0.0-wip` so we don't have to keep updating the hash?